src/universal/bin/evm-runner

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+java -cp "lib/*" io.iohk.ethereum.vmrunner.EvmRunner
```

Comment:
@ntallar so I ended up doing this instead of a separate SBT task. Seems like the easiest solution. We can change it later if it's not universal enough.